app/api/logging/flask_logger.py

```diff
@@ -102,6 +108,7 @@ def _log_end_request(response: flask.Response) -> flask.Response:
             "response.content_length": response.content_length,
             "response.content_type": response.content_type,
             "response.mimetype": response.mimetype,
+            "duration": datetime.now() - flask.g.request_start_time
```

Comment:
nit: i think API response times are most useful in milliseconds rather than seconds, so maybe let's report milliseconds elapsed and name 

nit: could we name this "response.time_ms" instead of "duration"?

Reply:
Changed